### DISCUSSION TITLE
ERROR ethers_solc::resolver: failed to resolve versions

### DISCUSSION THREAD

**Discussion by Killpit** [foundry-rs/foundry#6721] [Help]
I have an issue and even though added version to foundry.toml, still it doesn't work abd get the underlying error. Tried searching for the error but still couldn't solve it

![Ekran Resmi 2024-01-07 17 41 09](https://github.com/foundry-rs/foundry/assets/57031187/084d540e-b787-425d-ad34-2252b061b287)

**Comment by yehoudaB** [comment]
try to write instead 

```
solc_version = '0.8.21'
```
because your Counter.sol requires a version superior to 0.8.20
